### PR TITLE
Use subprocess.run for running run_bot.py

### PR DESCRIPTION
**Pull request by oliverchang**
Rather than mozprocess.

### CODE REVIEW

**Comment by oliverchang** [comment]
PTAL. speculative fix for https://github.com/google/clusterfuzz/issues/2007. On the affected bots, mozprocess is deadlocked waiting for a run_bot.py which no longer exists.

**Review by mbarbella-chromium — APPROVED** [review]

File: src/python/system/process_handler.py [hunk 1]
```diff
@@ -185,14 +185,14 @@ def run_process(cmdline,
...
-    cmd, args = shell.get_command_and_arguments(cmdline)
+    cmd = shell.get_command(cmdline)
 
     process_output = mozprocess.processhandler.StoreOutput()
     process_status = ProcessStatus()
     try:
       process_handle = mozprocess.ProcessHandlerMixin(
           cmd,
-          args,
+          None,
```
Comment (inferno-chromium) [inline src/python/system/process_handler.py:195]: args=None

File: src/python/bot/startup/run.py [hunk 2]
```diff
@@ -46,31 +44,28 @@
...
-  exit_code = process_handle.wait(timeout=run_timeout)
-  try:
-    process_handle.kill()
-  except Exception:
-    pass
-
-  log_message = (
-      'Command: %s %s (exit=%s)\n' % (command, arguments, exit_code) +
-      b'\n'.join(store_output.output).decode('utf-8', errors='ignore'))
+  log_message = ('Command: %s (exit=%s)\n' % (command, exit_code) +
```
Comment (inferno-chromium) [inline src/python/bot/startup/run.py:67]: nit:
output = output.decode('utf-8', errors='ignore')
Command: {command} (exit={exit_code})\n{output}

**Review by inferno-chromium — APPROVED** [review]

File: src/python/bot/startup/run.py (hunk 2 above)
Comment (oliverchang) [reply src/python/bot/startup/run.py:67]: Done!

File: src/python/system/process_handler.py (hunk 1 above)
Comment (oliverchang) [reply src/python/system/process_handler.py:195]: Done!

**Pull request merged**